content/advent-2016/saga-go-dependency-management.md

```diff
@@ -0,0 +1,106 @@
++++
+date = "2016-12-13T00:00:00"
+author = [ "Sam Boyer (@sdboyer)" ]
+title = "The Saga of Go Dependency Management"
+series = ["Advent 2016"]
++++
+
+# The Saga of Go Dependency Management
+
+The Go community is on the cusp of some major shifts in the way we handle dependencies. These shifts are a long time coming, and involve the work and effort of dozens, if not hundreds, of people. Six to twelve months from now, the Go dependency management landscape is likely to look very different.
+
+For those who haven't been following closely - or, honestly, even for those who have - it can be difficult to keep track of what's going on, and where things are going. So, as part of the Gopher advent series, we thought we'd tie up Go package management past, present, and future in a nice big bow.
+
+## The road so far
+
+Anyone who takes a stroll down the memory lane of Go package management will find ample clumps of ripped-out hair strewn about. The process has been long and frustrating, and there's far too much to cover in detail here. But the history is important to where we are now, so a high-level timeline is worth it.
```

Comment:
"important to where" => "important to understand where". I'd also remove "so a high level timeline is worth it" and just leave the first half of that sentence.